src/lib/utils.js

```diff
@@ -18,4 +19,31 @@ export function formatDate(date) {
       day = '0' + day;
 
   return [year, month, day].join('-');
-};
\ No newline at end of file
+};
+
+// YYYY-MM-DD string to YYYY-MM-DD date without time zone (aka in UTC)
+export function stringToDate(dateString) {
+   // Split the input string into components
+   const parts = dateString.split('-');
+   const year = parseInt(parts[0], 10);
+   const month = parseInt(parts[1], 10) - 1; // Months are 0-based in JavaScript Date
+   const day = parseInt(parts[2], 10);
+ 
+   // Create a Date object in UTC
+   const date = new Date(year, month, day);
+   return date;
```

Comment:
```suggestion
      return toUTCDate(date)
```

Reply:
ah comment is misleading, should return the YYYY-MM-DD date at 00:00, user's time zone